policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/applicable_amount.yaml

```diff
@@ -91,9 +96,11 @@ metadata:
   breakdown:
```

Comment:
can you apply the uprating in the general metadata and use `propagate_metadata_to_children`